README.md

```diff
@@ -1,16 +1,46 @@
-# esp32cam-edgelight
-ESP32 Camera sample app for Groundlight API
+<p align="center">
+  <a href="https://nextjs-fastapi-starter.vercel.app/">
+    <img src="https://avatars.githubusercontent.com/u/118213576?s=200&v=4" height="96">
+    <h3 align="center">ESP32CAM Edgelight Client</h3>
+  </a>
+</p>
 
-The camera will take pictures on defined intervals, send to the Groundlight.ai API over wifi, and send a notification via Slack when the camera first answers "YES", and then as the state changes.
+<br/>
 
-On first boot the device will launch access point to set wifi credentials, detector, slack hook, etc.  Holding a button during reset or power-on will force an erase of the file system and reset this process.   Serial port is configured to 115200 baud for debug messages.  
+# What is the ESP32CAM Edgelight Client?
 
-Works well with M5Stack ESP32 Camera, ESP32CAM, and likely others but would need to verify and update the pinouts.  Some hardware devices are more reliable than others and the software makes regular attempts to reboot as necessary which often gets things going again.
+The ESP32CAM Edgelight Client is a firmware for the ESP32 microcontroller that allows it to upload images to the Groundlight API.
```

Comment:
To be a bit more specific and friendly to the uninitiated user, you might say "...firmware for the ESP32 microcontroller that allows it to submit computer vision image queries using the Groundlight API" or something to that effect. Simply saying "upload images to..." seems to be missing the point.